draft-westerlund-tsvwg-sctp-crypto-chunk.md

```diff
@@ -177,15 +177,17 @@ protection buffer shall never exceed the SCTP payload size thus
 protection engine shall be aware of the PMTU (see {{pmtu}}).
 
 The key management part of the protection engine is the set of data and procedures
-that take care of key distribution, verification, and update.
+that take care of key distribution, verification, and update. For key management
+the Protection Engines exploit SCTP DATA chunks having a dedicated Payload
+Protocol Identifier.
 
 Key management of protection engine is RECOMMENDED to use the SCTP
-CRYPTO chunk for handshaking, in that case any packet being exchanged
-between protection engine peers shall be transported as payload of
-Crypto chunk (see {{crypto-chunk}}).
+DATA chunks inside SCTP CRYPTO chunk for handshaking, in that case
+any packet being exchanged between protection engine peers shall be
+transported as payload of Crypto chunk (see {{crypto-chunk}}).
```

Comment:
So, that this is true as soon one has reached "Established" state is fine. But what about before that? Should it use just plain text DATA chunks, however only the key-management PPID is allowed to be sent by the upper layers? Or should we like SCTP-AUTH have a null key? I think just using DATA until Established for the key-managment messages is the better operations and avoids opening up for using non-encrypted when you have reached established state.